lib/modules/modules.go

```diff
@@ -379,6 +383,16 @@ func (p *defaultModules) BuildType() string {
 	return teleportBuildType
 }
 
+// IsEnterpriseBuild returns if the binary was built with enterprise modules
```

Comment:
```suggestion
// IsEnterpriseBuild returns false for [defaultModules].
```

Or ditch the comment, it's a private type anyway. Similarly for others.